pkg/devicerepository/store/profiles.go

```diff
@@ -64,6 +64,20 @@ type EndDeviceProfile struct {
 	Supports32BitFCnt         bool                  `yaml:"supports32bitFCnt" json:"supports_32_bit_f_cnt"`
 }
 
+// EndDeviceProfileV2 is the profile of a LoRaWAN end device that is mentioned in the vendor index file
+// instead of being embedded in the end device profile.
+type EndDeviceProfileV2 struct {
+	VendorProfileID string `yaml:"vendorProfileID"`
+	// First profile format.
+	EndDeviceID     string `yaml:"endDeviceID"`
+	FirmwareVersion string `yaml:"firmwareVersion"`
+	HardwareVersion string `yaml:"hardwareVersion"`
+	Region          string `yaml:"region"`
+	// Second profile format.
+	ID    string `yaml:"id"`
+	Codec string `yaml:"codec"`
+}
```

Comment:
It's technically not a profile. These are identifiers to a profile.

The ID in the bottom is in fact a ProfileID.

Reply:
Renamed it.